service/src/test/java/bio/terra/pipelines/dependencies/gcs/GcsServiceTest.java

```diff
@@ -87,6 +87,28 @@ void generatePutObjectSignedUrl() {
     assertEquals(objectName, blobInfo.getName());
   }
 
+  @Test
+  void generateResumablePostObjectSignedUrl() {
```

Comment:
is there any way to test for something specific to this type of signedUrl like the header?  I assume no since we didnt do it on the PUT signed url test but just thought id ask.